src/ducks/transactions/actions/HealthExpenseStatusAction.jsx

```diff
@@ -169,11 +167,7 @@ const action = {
     return <Icon icon="hourglass" color={color} />
   },
   match: transaction => {
-    return (
-      isHealthExpense(transaction) &&
-      getVendors(transaction) &&
-      !flag('reimbursements.tag')
-    )
+    return false
```

Comment:
This action is not used anymore, replaced by ReimbursementStatusAction. This file can be removed.

Reply:
https://github.com/cozy/cozy-banks/issues/1754